### ISSUE TITLE
[POI] Support scenarios related to a user's home or work address

### ISSUE THREAD

**Issue by ryanisgrig** [microsoft/botframework-solutions#2218]
## User Story
As ... a bot user
I want ... to find points of interest around my home or work locations
so that ... the experience is personalized to me

## Acceptance Criteria 
*List your requirements.*
- [ ] Requires user profile store feature
- [ ] Fill in slots for both home/work locations in POI skill
- [ ] LUIS model needs to support entities matching both home and work utterances (home, house, office, work, etc.)
- [ ] When home/work entity is uttered, use the coordinates of corresponding location to
- [ ] If slots are not filled (user profile is not enabled/passing through locations), ask user for their home/work location

## Resources
*Provide any screenshots or conversation mockups to support your request.*
*Learn how to mock up bot conversations with [.chat files](http://aka.ms/bfchatfile).*

##### When home/work slot is known
```
User: Find a coffee shop near my house
Bot: I found a few coffee shops near your home
```

```
User: Find a coffee shop near work
Bot: I found a few coffee shops near your office
```

```
User: Take me home
Bot: *start navigation to home coordinates*
```

```
User: Take me to work
Bot: *starts navigation to work coordinates*
```

##### When home/work slot is unknown

```
User: Find a coffee shop near my house
Bot: I don't have your home address, where is that?
User: 123 Sesame Street
Bot: Ok got it. *Send back to VA*
Bot: I found a few coffee shops near your home
```

```
User: Find a coffee shop near work
Bot: I don't have your work address, where is that?
User: 123 Sesame Street
Bot: Ok got it. *Send back to VA*
Bot: I found a few coffee shops near your office
```

```
User: Take me home
Bot: I don't have your home address, where is that?
User: 123 Sesame Street
Bot: Ok got it. *Send back to VA*
Bot: *start navigation to home coordinates*
```

```
User: Take me to work
Bot: I don't have your work address, where is that?
User: 123 Sesame Street
Bot: Ok got it. *Send back to VA*
Bot: *starts navigation to work coordinates*
```

## Notes
*Provide any additional information.*
This is currently blocked by a user profile store at #1362

**Comment by darrenj** [comment]
Not in scope for PoI moving forward. We can revisit needs for this end to end when we have a customer use for the skill
